### PR TITLE
Update MegaGlest.org.xml

### PR DESCRIPTION
**Pull request by jeremyn**
This pull request replaces pull request https://github.com/EFForg/https-everywhere/pull/13769.

### CODE REVIEW

File: src/chrome/content/rules/MegaGlest.org.xml [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
 	<target host="www.megaglest.org" />
+	<target host="archer.megaglest.org" />
+	<target host="ci.megaglest.org" />
 	<target host="docs.megaglest.org" />
+	<target host="filux.megaglest.org" />
 	<target host="forum.megaglest.org" />
-
-	<target host="faq.megaglest.org" />
-	<target host="readme.megaglest.org" />
-	<target host="wiki.megaglest.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/MegaGlest.org.xml]: Why are you removing these redirects?

Comment (jeremyn) [reply src/chrome/content/rules/MegaGlest.org.xml]: I wrote that in https://github.com/EFForg/https-everywhere/pull/13769#issuecomment-347211172:

>@heuser443 I get what you're doing here, but I think this is the wrong approach. These sort of deep redirects, like http://translate.megaglest.org to https://docs.megaglest.org/MG/Translations , are generally bad practice. We've also caused a specific problem for this site.
>
>Let's just remove all `target`s and `rule`s that don't support a direct rewrite. We'll lose some security, but I think that's fine here. The site's users and owners can work toward supporting HTTPS better if they want to.

Comment (J0WI) [reply src/chrome/content/rules/MegaGlest.org.xml]: Okay, that sounds reasonable to me.

**Comment by cschanaj** [comment]
this should close #13765

**Comment by jeremyn** [comment]
Thanks @cschanaj , I've closed that issue.

**Pull request merged**